plugins/scaffolder-node/api-report.md

```diff
@@ -16,13 +15,19 @@ import { Writable } from 'stream';
 import { z } from 'zod';
 
 // @public
-export type ActionContext<TActionInput extends JsonObject> = {
+export type ActionContext<
+  TActionInput extends JsonObject,
+  TActionOutput extends JsonObject = any,
+> = {
   logger: Logger;
   logStream: Writable;
   secrets?: TaskSecrets;
   workspacePath: string;
   input: TActionInput;
-  output(name: string, value: JsonValue): void;
+  output<KEY extends keyof TActionOutput>(
```

Comment:
I wonder if we can keep the old `output(name: string, value: JsonValue): void` if `TOutputSchema` doesn't extend `Zod` so that we can remove the `any`'s and not break the old behaviour.